docs/styles/spec/file-input-spec.less

```diff
@@ -1,5 +1,5 @@
 @import (reference) 'components/button/index';
-@import (reference) 'components/file/index';
+//@import (reference) 'components/file/index';
```

Comment:
Change to...

```less
@import (reference) 'components/file/hx-file-control';
```

^^ that should allow us to remove `components/file/index.less` but still allow the spec styles to generate as expected.

Reply:
you _may_ need to import `mixins`, but I'm not sure

Reply:
@lalithkarikelli, please see comment above and verify there aren't any errors:

change:
```less
//@import (reference) 'components/file/index';
```

to:
```less
@import (reference) 'components/file/hx-file-control';
```